tools/releaser/breaking_changes/v20240530002.md

```diff
@@ -0,0 +1,54 @@
+# Breaking Changes
+
+## SDK changes
+
+go.mongodb.org/atlas-sdk/v20240530002/admin
+Incompatible changes:
+
+- (\*DiskBackupExportJob).GetComponents: changed from func() []DiskBackupBaseRestoreMember to func() []DiskBackupExportMember
+- (_DiskBackupExportJob).GetComponentsOk: changed from func() (_[]DiskBackupBaseRestoreMember, bool) to func() (\*[]DiskBackupExportMember, bool)
+- (\*DiskBackupExportJob).SetComponents: changed from func([]DiskBackupBaseRestoreMember) to func([]DiskBackupExportMember)
+- (*InvoicesApiService).ListInvoicesExecute: changed from func(ListInvoicesApiRequest) (*PaginatedApiInvoice, *net/http.Response, error) to func(ListInvoicesApiRequest) (*PaginatedApiInvoiceMetadata, \*net/http.Response, error)
+- (\*LiveMigrationRequest).GetDropEnabled: removed
+- (\*LiveMigrationRequest).GetDropEnabledOk: removed
+- (\*LiveMigrationRequest).SetDropEnabled: removed
+- AuthorizeCloudProviderAccessRoleApiParams.CloudProviderAccessAWSIAMRoleUpdateRequest: removed
+- CloudProviderAccessAWSIAMRoleRequest: removed
+- CloudProviderAccessAWSIAMRoleUpdateRequest: removed
+- CreateCloudProviderAccessRoleApiParams.CloudProviderAccessAWSIAMRoleRequest: removed
+- CreatePushMigrationApiParams.LiveMigrationRequest: removed
+- DiskBackupBaseRestoreMember: removed
+- DiskBackupExportJob.Components: changed from _[]DiskBackupBaseRestoreMember to _[]DiskBackupExportMember
+- InvoicesApi.ListInvoicesExecute: changed from func(ListInvoicesApiRequest) (*PaginatedApiInvoice, *net/http.Response, error) to func(ListInvoicesApiRequest) (*PaginatedApiInvoiceMetadata, *net/http.Response, error)
+- ListInvoicesApiRequest.Execute: changed from func() (*PaginatedApiInvoice, *net/http.Response, error) to func() (*PaginatedApiInvoiceMetadata, *net/http.Response, error)
+- LiveMigrationRequest.DropEnabled: removed
+- LiveMigrationRequest: removed
+- NewCloudProviderAccessAWSIAMRoleRequest: removed
+- NewCloudProviderAccessAWSIAMRoleRequestWithDefaults: removed
+- NewCloudProviderAccessAWSIAMRoleUpdateRequest: removed
+- NewCloudProviderAccessAWSIAMRoleUpdateRequestWithDefaults: removed
+- NewDiskBackupBaseRestoreMember: removed
+- NewDiskBackupBaseRestoreMemberWithDefaults: removed
+- NewLiveMigrationRequest: removed
+- NewLiveMigrationRequestWithDefaults: removed
+- ValidateMigrationApiParams.LiveMigrationRequest: removed
```

Comment:
To be confirmed if changes do not affect Integrations. 
There is a large amount of openapi corrections that remove field here.

CC @lantoli